syntax = "proto3";

message CooperateResponse {
    bool accept = 1;
};
